bridge/engine/EngineStreamDirector.h

```diff
@@ -547,9 +635,28 @@ class EngineStreamDirector
         return false;
     }
 
+    void setSlidesSsrcAndBandwidth(size_t slidesSsrc, uint32_t bwKbps)
+    {
+        _slidesSsrc = slidesSsrc;
+        _slidesBandwidthKbps = bwKbps;
+    }
+
 private:
-    static const size_t lowQuality = 0;
-    static const size_t midQuality = 1;
+    static constexpr size_t lowQuality = 0;
```

Comment:
This looks like an enum.

Reply:
Made to enum, but with explicit assignemnt from 0 to 3. 'constexpr' is the modern way to express the same, when we realy care about compile-time assigned value (and we do care here, because we use this levels as indexes in the arrays.)